react/Footer/data/connect.js

```diff
@@ -8,7 +8,8 @@ export default [
     name: 'Work for SEEK',
     href: '/work-for-seek',
     analytics: 'toolbar:work+for+seek',
-    secondary: true
+    secondary: true,
+    badge: 'new'
```

Comment:
Are we supporting multiple badge types? Would this be better off as `new: true`?